test/Spaces/sphere.jl

```diff
@@ -5,36 +5,51 @@ import ClimaCore: Domains, Topologies, Meshes, Spaces, Geometry, column
 using Test
 
 @testset "Sphere" begin
-    FT = Float64
-    context = ClimaComms.SingletonCommsContext()
-    radius = FT(3)
-    ne = 4
-    Nq = 4
-    domain = Domains.SphereDomain(radius)
-    mesh = Meshes.EquiangularCubedSphere(domain, ne)
-    topology = Topologies.Topology2D(context, mesh)
-    quad = Spaces.Quadratures.GLL{Nq}()
-    space = Spaces.SpectralElementSpace2D(topology, quad)
+    for FT in (Float64, Float32)
+        context = ClimaComms.SingletonCommsContext()
+        radius = FT(3)
+        ne = 4
+        Nq = 4
+        domain = Domains.SphereDomain(radius)
+        mesh = Meshes.EquiangularCubedSphere(domain, ne)
+        topology = Topologies.Topology2D(context, mesh)
+        quad = Spaces.Quadratures.GLL{Nq}()
+        space = Spaces.SpectralElementSpace2D(topology, quad)
+
+        # surface area
+        @test sum(ones(space)) ≈ FT(4pi * radius^2) rtol = 1e11 * eps(FT)
+
+        enable_bubble = false
+        no_bubble_space =
+            Spaces.SpectralElementSpace2D(topology, quad; enable_bubble)
```

Comment:
```suggestion
        no_bubble_space =
            Spaces.SpectralElementSpace2D(topology, quad; enable_bubble = false)
```

Reply:
Sorry, I am just seeing your comments now, after this has been merged.